### PR TITLE
Database setup

### PR DESCRIPTION
**Pull request by Pocketzs**
- Initial database setup
- created initial migrations and models
- created csv loading rake tasks 
- Model Test coverage at 100%

Note: rails generator used for models, extra unused files may be present throughout app.

### CODE REVIEW

File: db/migrate/20230103220258_create_transactions.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreateTransactions < ActiveRecord::Migration[5.2]
+  def change
+    create_table :transactions do |t|
+      t.references :invoice, foreign_key: true
+      t.integer :credit_card_number, limit: 8
```
Comment (Pocketzs) [inline db/migrate/20230103220258_create_transactions.rb:5]: Very interesting find on how integers have byte limits

**Comment by jlweave** [comment]
All db setup was done during a zoom call with group present. Completed with driver navigator.

**Comment by JustJakeSeymour** [comment]
Since `g model` also made factories for us, I suggest we use `faker` to get those fully operational

File: db/migrate/20230103220258_create_transactions.rb (hunk 1 above)
Comment (ajongaro) [reply db/migrate/20230103220258_create_transactions.rb:5]: Yes! The 'limit' symbol triggers the bigint column type in Postgres.

**Review by JustJakeSeymour — APPROVED** [review]
Lot's went into this set up, and we will add more associations as needed. 

Ready for User Stories!

**Comment by ajongaro** [comment]
Looks good, great work team!

**Review by jlweave — APPROVED** [review]
completed in a group on zoom, driver navigator style.

**Review by ajongaro — APPROVED** [review]
Looks good! Onto Factory Bot testing.

**Pull request merged**